lib/order.rb

```diff
@@ -37,27 +56,17 @@ def calculate_total(order)
   end
 
   def prints_order_summary(order)
```

Comment:
I also had this as one method originally but then thought maybe broken into 'summary of items' and 'summary of price' would help with the 'one method does one thing' idea. Then you could write another method which calls one then the other if that makes sense